### PR TITLE
NUnit Console 3.12 release notes

### PR DESCRIPTION
**Pull request by ChrisMaddock**
Release notes for NUnit Console 3.12. Currently planning to do the release tomorrow, will take this out of draft status when it's out. 🙂

### CODE REVIEW

File: docs/articles/nunit/release-notes/console-and-engine.md [hunk 1]
```diff
@@ -1,5 +1,36 @@
...
+
+## .NET Core NUnit Console 3.12 Beta 2  - January 17, 2021
+
+This release contains various improvements to running tests on .NET Core and Mono, and changes to
+extension loading logic to allow the Engine to better support extensions which target multiple platforms. There are additionally
+a number of fixes to issues that were identified with 3.12 Beta 1. Please also be aware that this will be the last version of
+the NUnit Engine to support .NET Standard 1.6.
+
+The .NET Core Console remains in Beta due to some unresolved dependency loading and framework targeting issues - contributions to
+fix this issues would be very welcome!
```
Comment (mikkelbu) [inline docs/articles/nunit/release-notes/console-and-engine.md:13]: Suggestion
```suggestion
fix these issues would be very welcome!
```

**Review by mikkelbu — APPROVED** [review]
Only a single suggestion

File: docs/articles/nunit/release-notes/console-and-engine.md (hunk 1 above)
Comment (ChrisMaddock) [reply docs/articles/nunit/release-notes/console-and-engine.md]: Thanks again! Serves me right for copy/pasting. 🙂

**Comment by ChrisMaddock** [comment]
Thanks both, merging. 🙂

**Pull request merged**